docs/deployment/migration-guide.md

```diff
@@ -17,14 +17,18 @@
 
 # Kyuubi Migration Guide
 
+## Upgrading from Kyuubi 1.7.0 to 1.7.1
+
+* Since Kyuubi 1.7.1, the REST API of `Open(create) a session` will not contain parameters `protocolVersion`.
```

Comment:
I think we should clear document it does not break user to use old REST client to connect Kyuubi Server.
```
Since Kyuubi 1.7.1, `protocolVersion` is removed from the request parameters of the REST API `Open(create) a session`. All removed or unknown parameters will be silently ignored and affects nothing.
```